Assets/Scripts/Player.cs

```diff
@@ -37,7 +39,8 @@ void FixedUpdate()
     {
         Drive();
         Nitro();
-        //Cook();
+        Steer();
+        Drift();
     }
 
     private void Update()
```

Comment:
I would add reading your inputs here and in your movement functions to use those inputs for direciton